app/views/users/_booking_request_list.html.erb

```diff
@@ -0,0 +1,15 @@
+ <hr>
+  <h2>Booking requests...</h2>
+  <% current_user.bookings.each do |booking| %>
+  <% if !booking.confirmed %>
+          <div class="d-flex justify-content-between align-items-center my-3">
```

Comment:
Line 5: Can you add a component for this in assets and move the styling in there?